goss-testing/scripts/monitoring_check.sh

```diff
@@ -12,10 +12,10 @@ do
     m_ns="$(echo $m | awk -F, '{print $1}')"
     m_url="$(echo $m | awk -F, '{print $2}' | sed 's/[][]//g' | tr -d '"')"
 
-    # Check for 307 redirects (location oauth and keycloak)
-    response=$(curl -i -s -S -L --max-redirs 2 https://$m_url/ | grep -c "HTTP/2 307")
+    # Check for 302 redirect from oauth2-proxy to the Keycloak login page
```

Comment:
302 is for a temporary move, is this a permanent redirect (should it be 301?)?

Reply:
I suppose it has to be 302 since it's a _test_ checking a _result_, and that result is 302... right?

Reply:
oauth2-proxy returns a 302 response, so that's what the test is checking for. It works with my browser.